docs/units/combining_and_defining.rst

```diff
@@ -99,3 +106,39 @@ can be enabled by calling :func:`~astropy.units.add_enabled_units`::
   ]
 
 .. EXAMPLE END
+
+.. _enabling_units:
+
+Example
+=======
+
+.. EXAMPLE START: Enabling New Units
```

Comment:
I'm not sure if there's much point in duplicating this example. It might be a good idea to only have the example written out in the Table documentation and to just link to that from here.